frontend/containers/for-public-pages/carrousel/component.tsx

```diff
@@ -0,0 +1,141 @@
+import { FC, useState } from 'react';
+
+import { ChevronLeft, ChevronRight } from 'react-feather';
+import { FormattedMessage } from 'react-intl';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+
+import { useBreakpoint } from 'hooks/use-breakpoint';
+
+import Button from 'components/button';
+import Icon from 'components/icon';
+import LayoutContainer from 'components/layout-container';
+import { Paths } from 'enums';
+
+import { PublicPageCarrouselProps } from './index';
+
+export const ForPublicPagesCarrousel: FC<PublicPageCarrouselProps> = ({
+  texts,
+  images,
+  subtitle,
+  title = <FormattedMessage defaultMessage="What HeCo Invest can do for you" id="2sU5tO" />,
+}) => {
+  const [selectedImageIndex, setSelectedImageIndex] = useState(0);
+
+  const breakpoint = useBreakpoint();
+  const isMd = breakpoint('sm');
+  const isXl = breakpoint('xl');
+
+  return (
+    <div className="py-20 mt-20 bg-green-dark">
+      <LayoutContainer>
+        <div className="flex flex-col items-center text-center text-white">
+          <h2 className="mb-6 font-serif text-3xl font-bold lg:text-4xl">{title}</h2>
+          <p className="max-w-xl mb-12 lg:text-lg">{subtitle}</p>
+          <div className="w-[calc(100vw-20px)] overflow-hidden bg-green-dark mb-0 lg:mb-12">
+            <div
+              className={cx(
+                'flex lg:gap-6 2xl:gap-10 w-[500vw] lg:w-[183vw] transition-all ease-in-out duration-700',
+                {
+                  'translate-x-0 lg:translate-x-24': selectedImageIndex === 0,
+                  'translate-x-[calc(-100vw+20px)] lg:translate-x-[calc(-33.33vw+88px)]':
+                    selectedImageIndex === 1,
+                  'translate-x-[calc(-200vw+40px)] lg:translate-x-[calc(-66.66vw+78px)]':
+                    selectedImageIndex === 2,
```

Comment:
Here we are assuming there are only three images, right? Could you comment that on top of the `images` property in the `types.ts` file?